src/Symfony/Component/HttpFoundation/Tests/Session/Storage/Handler/Fixtures/empty_destroys.expected

```diff
@@ -0,0 +1,17 @@
+open
+validateId
+read
+doRead: abc|i:123;
+read
+
+write
+destroy
+doDestroy
+close
+Array
+(
+    [0] => Content-Type: text/plain; charset=utf-8
+    [1] => Cache-Control: private, max-age=10800
```

Comment:
http://php.net/manual/en/function.session-cache-limiter.php says something about `pre-check`. So php documentation is wrong about that? pre-check doesn't seem to be a standard anyway.

Reply:
Looks like so yes...